src/Symfony/Component/Console/Tests/Input/InputTest.php

```diff
@@ -129,4 +129,12 @@ public function testSetGetInteractive()
         $input->setInteractive(false);
         $this->assertFalse($input->isInteractive(), '->setInteractive() changes the interactive flag');
     }
+
+    public function testSetGetStream()
+    {
+        $input = new ArrayInput(array());
+        $stream = fopen('php://memory', 'r+', false);
```

Comment:
even if there is no explicit checks, tests should respect the contract of the class.